src/codegen.zig

```diff
@@ -140,6 +140,29 @@ pub fn generateFunction(
     }
 }
 
+fn writeFloat(comptime F: type, f: F, target: Target, endian: std.builtin.Endian, code: []u8) void {
+    if (F == f80) {
```

Comment:
I think this should be handled same as every other float.

Reply:
As below, the source is here: https://github.com/ziglang/zig/blob/7cfc3f0cfa626abe25c8318a7852977cbc1c723b/src/value.zig#L1273 I actually don't really know yet how to handle `f80` at all. I can leave is a todo instead then.

Reply:
`f80` is now explicitly unhandled in b0bb158. Happy to come back to this discussion when I start working on floats in x64. Until then, lowering floats was an unfortunate prerequisite to not regress comptime tests it seems.